src/main/java/it/gov/pagopa/pu/pagopapayments/connector/PagoPaApiClient.java

```diff
@@ -0,0 +1,51 @@
+package it.gov.pagopa.pu.pagopapayments.connector;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.gov.pagopa.pu.pagopapayments.dto.PaTaxonomyDTO;
+import it.gov.pagopa.pu.pagopapayments.exception.ApplicationException;
+import it.gov.pagopa.pu.pagopapayments.util.RestUtil;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.List;
+
+@Service
+@Slf4j
+public class PagoPaApiClient {
+
+  private final RestTemplate restTemplate;
+  private final ObjectMapper objectMapper;
+  private final String baseUrlGithub;
+  private final String taxonomyPathContext;
+
+  public PagoPaApiClient(
+    @Value("${rest.pagopa-platform.base-url}") String baseUrlGithub,
+    @Value("${rest.pagopa-platform.taxonomy.service.context}") String taxonomyPathContext,
```

Comment:
Let's not split these properties.
Let's keep it simple, let's just have one string having the complete URL instead:
`rest.pagopa-taxonomies.base-url`